src/main.qml

```diff
@@ -517,4 +517,130 @@ ApplicationWindow
             }
         }
     }
+
+    Component.onCompleted: {
+        var query = readCLIArguments(Qt.application.arguments,":");
+        var addr = "";
+
+        if ("parameters" in query) {
```

Comment:
Minor, maybe done on one line `"parameters" in query && "address" in query.parameters`.